deployment/modules/1password/account/k8s-secrets.tf

```diff
@@ -25,3 +25,37 @@ resource "onepassword_item" "containerssh_host_key" {
     }
   }
 }
+
+resource "random_password" "grafana_admin_user" {
+  length           = 40
+  special          = true
+  override_special = "!@#$%^&*()_+"
+}
+
+resource "onepassword_item" "grafana_admin_user" {
+  vault    = data.onepassword_vault.kubernetes.uuid
+  title    = "grafana-admin-user"
+  category = "password"
+  password_recipe {
```

Comment:
You have a password recipe here so this will automatically generate a password through 1password, so you probably don't need the random generated password above